lib/workers/repository/onboarding/common.ts

```diff
@@ -0,0 +1,31 @@
+import hasha from 'hasha';
+import { configFileNames } from '../../../config/app-strings';
+import type { RenovateConfig } from '../../../config/types';
+import { logger } from '../../../logger';
+
+export function defaultConfigFile(config: RenovateConfig): string {
+  return configFileNames.includes(config.onboardingConfigFileName!)
+    ? config.onboardingConfigFileName!
+    : configFileNames[0];
+}
+
+export class OnboardingState {
```

Comment:
I'm thinking about this class. still not sure if this is the best way to handle static state for a repo. 🤔

Reply:
other options I can think of are an exported plain constant (but than we lose the tracing, which is the only benefit of the static class) and maybe `memCache`?

Reply:
ok, leave this class but store the value inside memcache, so we ensure reset after each repo.